### PR TITLE
C Code Formatting

### PR DESCRIPTION
**Pull request by robodhruv**
_New Line was not being accpeted._

Random!

### CODE REVIEW

**Comment by udiboy1209** [comment]
does this change syntax highlight correctly? I had tried this last time but it hadn't worked

**Comment by robodhruv** [comment]
Yep. On my system, as well as on my blog. This works.

Just to make it certain, I added _2 spaces_ and return after each line in code. Works well! :)
par Travis kyu fail ho raha hai? I hardly changed anything!

File: _posts/2016-03-12-remote-controlling-a-bot-with-android.md [hunk 1]
```diff
@@ -47,20 +47,20 @@ AVR Code:
 
 Code is pretty straight forward too. Take the bit-sequence from HC-05 through UART and output it on the four pins of PORTB. I have posted the relevant `main()` function here. Entire code can be found on github.
 
-{% highlight c %}
+```c
```
Comment (udiboy1209) [inline _posts/2016-03-12-remote-controlling-a-bot-with-android.md:50]: This is the change which messes up syntax highlighting

**Comment by robodhruv** [comment]
Yes, I see it. :p

build offline and check if it works. seems to be working fine for me :)

**Pull request closed without merging**